control-plane/controllers/configentries/configentry_controller_test.go

```diff
@@ -543,48 +544,59 @@ func TestConfigEntryControllers_createsConfigEntry(t *testing.T) {
 	}
 
 	for _, c := range cases {
-		t.Run(c.kubeKind, func(t *testing.T) {
-			req := require.New(t)
-			ctx := context.Background()
+		for _, secure := range []bool{true, false} {
```

Comment:
updated these tests to run in secure and non secure mode to ensure config entries are created when acls are enabled